src/stavenote.js

```diff
@@ -81,7 +81,7 @@ export class StaveNote extends StemmableNote {
       let minL = props[props.length - 1].line;
```

Comment:
Would this work for tab notes too?

Reply:
I don't think they have ledger lines.  If you're thinking of the stem-length PR it might, but I they won't need it because of lack of ledger lines.